packages/features/portfolio/src/hooks/use-vault-holdings-table.tsx

```diff
@@ -224,6 +235,20 @@ export const useVaultHoldingsTable = () => {
       const totalPoints =
         arbPoints?.find(({ token }) => token === v.vaultShares.tokenId)
           ?.points || 0;
+      const vaultReinvestmentData = reinvestmentData?.[v.vaultAddress];
+      let arbReinvestmentDate = '';
+      let mainnetReinvestmentDate = '';
+
+      if (vaultReinvestmentData) {
```

Comment:
Move the code inside this if block into a separate helper function so we can easily update the reinvestment cadences.